projects/storefrontlib/src/cms-structure/services/cms-components.service.ts

```diff
@@ -160,7 +175,8 @@ export class CmsComponentsService {
    * Return DeferLoadingStrategy for component type.
    */
   getDeferLoadingStrategy(componentType: string): DeferLoadingStrategy {
```

Comment:
Type of the function return to update (strict mode)

Reply:
👍